tests/src/JIT/HardwareIntrinsics/X86/Shared/SimpleBinOpTest.template

```diff
@@ -52,6 +52,12 @@ namespace JIT.HardwareIntrinsics.X86
                 // Validates passing a static member works
                 test.RunClsVarScenario();
 
+                if ({LoadIsa}.IsSupported)
```

Comment:
@CarolEidt, are these roughly the additional scenarios you were thinking of? Are there others you would like me to add before I go updating all the templates?

Reply:
Yes, this is what I had in mind. I can't think of others at the moment.